### PR TITLE
API link page

### LINKED ISSUE #65: References > Links to Existing API
Include links out to existing API Documentation (temporary solution pending OpenAPI)
- [ ] [PlaceOS Core API](https://placeos.docs.apiary.io/#)
- [ ] [Staff API](https://placeoscalendar.docs.apiary.io/#)

### PR DESCRIPTION
**Pull request by mghill**
Very minimal stub to carry links out to apiary

### CODE REVIEW

File: reference/api.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: API Links
+description: External links to PlaceOS Apiary
```
Comment (kimburgess) [inline reference/api.md]:
```suggestion
title: API
description: Specifications for interaction with PlaceOS
```

File: reference/api.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+---
+
+For detailed information on PlaceOS API, please see them on Apiary:
+
+- [PlaceOS API](https://placeos.docs.apiary.io/#)
+
+- [PlaceOS Staff App API](https://placeoscalendar.docs.apiary.io/#)
```
Comment (kimburgess) [inline reference/api.md:10]: I've heard the referred to lately as both `Staff API` or `Workplace API`. @jeremyw24 can you confirm correct terminology?

File: reference/api.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+title: API Links
+description: External links to PlaceOS Apiary
+---
+
+For detailed information on PlaceOS API, please see them on Apiary:
```
Comment (kimburgess) [inline reference/api.md]: The fact these are on Apiary is not important information.

File: reference/api.md (hunk 2 above)
Comment (jeremyw24) [reply reference/api.md:10]: @kimburgess I believe its still Staff API

File: reference/api.md (hunk 3 above)
Comment (jeremyw24) [reply reference/api.md]: Agreed @mghill this can be removed.

**Comment by jeremyw24** [comment]
Resolve Kim's comments otherwise LGTM

File: reference/api.md (hunk 3 above)
Comment (mghill) [reply reference/api.md]: Changed, but still notes they are external links, rather than to pages within the doc site, as this is best practice.

**Pull request merged**